cli/defenseclaw/tui/services/v8_event_history.py

```
@@ -190,11 +603,37 @@ def _decode_v8_event_history_rows(rows: list[tuple[Any, ...]]) -> tuple[V8EventH
                 outcome=outcome,
                 payload=payload,
                 payload_truncated=int(row[21] or 0) > _MAX_PAYLOAD_BYTES,
+                finding_tags=finding_tags,
+                hook_decision=str(row[24] or ""),
             )
         )
     return tuple(result)
 
 
+def _decode_finding_tags(raw_value: Any, raw_length: Any) -> tuple[str, ...]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `raw_value`

(ANN401)